### DISCUSSION TITLE
how do i remove the runners which are in offline state in one shot  at enterprise level

### DISCUSSION THREAD

**Discussion by swathida-byte** [actions/actions-runner-controller#1818] [Questions]

**Comment by allenb-epilog** [comment]
If you click on the runner name you can then force remove them within the runner details page. The main runner page wasn't letting me delete them for an organization, it would silently fail.

**Reply by hunjixin** [reply]
not work . force remove .but create this pod again

**Comment by samus-aran** [comment]
We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.
